### DISCUSSION TITLE
Token creating issue

### DISCUSSION THREAD

**Discussion by olga-density** [gravitational/teleport#30818] [Q&A] [answered]
Hello!

I have an issue with creating token:

```
tctl tokens add --ttl=5m --type=node
ERROR: creating token
unknown method CreateToken for service proto.AuthService

tctl version
Teleport v13.3.4 git: go1.21.0
teleport version
Teleport v13.3.4 git: go1.21.0
```
Any ideas/tips/help would be very much appreciated.

**Comment by zmb3** [comment, marked as answer]
Typically this means the version of `tctl` you are using is newer than the version of teleport that the cluster is running.

You might also want to try using our official builds from goteleport.com/download. I can tell you're not using them because we haven't completed our load testing with Go 1.21 yet 🙂

**Reply by olga-density** [reply]
appreciate your prompt response. I used official builds and I was able to create a new token. Thank you!
